src/events/gamebanana_watcher.js

```diff
@@ -0,0 +1,144 @@
+require('dotenv').config();
+const { MessageEmbed } = require('discord.js');
+const fetch = require('node-fetch');
+
+module.exports = {
+    name: 'ready',
+    execute: async (client) => {
+        let cssMaps = [];
+        let csgoMaps = [];
+        const cssUrl =
+            'https://gamebanana.com/mods/cats/5568?api=SubmissionsListModule';
+        const csgoUrl =
+            'https://gamebanana.com/mods/cats/5422?api=SubmissionsListModule';
+        let guild = client.guilds.resolve(process.env.GUILD_ID);
+        guild.channels
+            .fetch(process.env.NEW_MAPS_CHANNEL_ID)
+            .then((GuildChannel) => {
+                GuildChannel.fetch().then((channel) => {
+                    setInterval(async () => {
+                        fetch(cssUrl)
+                            .then((data) => data.text())
```

Comment:
```suggestion
                            .then((data) => data.json())
```
To access the JSON data directly.